dlrover/trainer/torch/elastic_dataloader.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from torch.utils.data import BatchSampler, DataLoader
+
+
+class ElasticDataLoader(DataLoader):
+    def __init__(self, *args, config_file=None, **kwargs):
+        super(ElasticDataLoader, self).__init__(*args, **kwargs)
+        self.current_batch_size = self.batch_size
+        self.config_file = config_file
+
+        if self.config_file:
+            self.load_config()
+
+    def load_config(self):
```

Comment:
```
def load_config(self, config_file=None):
    if not config_file:
        return
    with open(config_file, "r") as f:
       ...
```
The refactor can reduce indent levels and we and test the `local_config` without changing the class.